game_api.txt

```diff
@@ -258,7 +258,7 @@ The doors mod allows modders to register custom doors and trapdoors.
 	material = "default:wood",
 	groups = {choppy = 2, oddly_breakable_by_hand = 2, flammable = 2},
 	sounds = default.node_sound_wood_defaults(), -- optional
-	on_rightclick = function(pos, node, clicker, itemstack, pointed_thing) 
+	on_rightclick = function(pos, node, clicker, itemstack, pointed_thing)
```

Comment:
All the changes above this point show no visible change, are these deletion of whitespace?

Reply:
Yes, see for example https://github.com/minetest/minetest_game/pull/2624/files?w=1